mathics/__init__.py

```diff
@@ -8,23 +8,6 @@
 import numpy
 
 from mathics.version import __version__
-from mathics.core.symbols import Symbol
-from mathics.core.atoms import (
```

Comment:
I believe these imports are allowing a lot of sloppiness in imports. 

Right now we don't have any non-mathics library uses. Although something like this down the line might make sense, I think we should come back to this after the code has been reorganized.